### PR TITLE
spotify.py version 0.9: Add support for oauth keys being stored in secure data

### PR DESCRIPTION
**Pull request by butlerx**
I've emailed xt as script owner, waiting for a response

### CODE REVIEW

File: python/spotify.py [hunk 1]
```diff
@@ -43,48 +45,66 @@
...
-    "buffers"       : 'A comma separated list of buffers the script should check',
-    "emit_notice"   : 'If on, this script will use /notice, if off, it will use /msg to post info',
-    "client_id"     : 'required client id token go to https://developer.spotify.com/my-applications/#!/applications to generate your own',
-    "client_secret" : 'required client secret token go to https://developer.spotify.com/my-applications/#!/applications to generate your own'
+    "buffers":
+    'A comma separated list of buffers the script should check',
+    "emit_notice":
+    'If on, this script will use /notice, if off, it will use /msg to post info',
+    "client_id":
+    'required client id token go to https://developer.spotify.com/my-applications/#!/applications to generate your own',
```
Comment (Zarthus) [inline python/spotify.py]: What was your line limit restriction when reformatting the code? If it was 80 this exceeds said limit.

File: python/spotify.py [hunk 2]
```diff
@@ -43,48 +45,66 @@
...
+    "client_secret":
+    'required client secret token go to https://developer.spotify.com/my-applications/#!/applications to generate your own'
```
Comment (Zarthus) [inline python/spotify.py]: Same as above.

**Comment by Zarthus** [comment]
I think if you're going to be reformatting code it would be better to do it in separate commits. It's a pain to review otherwise.

**Comment by butlerx** [comment]
sorry I didn't mean to actually reformat the code. my editor had a autopep8 function i didnt notice was run

**Comment by flashcode** [comment]
Merged, thanks.

**Pull request closed without merging**